packages/payments-service/src/payment-request/provider.ts

```diff
@@ -0,0 +1,39 @@
+import { DataMapper } from '@aws/dynamodb-data-mapper'
+import { Injectable } from '@nestjs/common'
+import { currencySymbolMapper } from '@reapit/utils-common'
+import { defaultEmailConfig, sendEmail } from '../core/ses-client'
+import { PaymentRequestDto, PaymentRequestHeaders } from './dto'
+import { paymentRequestTemplate } from './template'
+
+@Injectable()
+export class PaymentRequestProvider {
+  constructor(private readonly datamapper: DataMapper) {}
+
+  async create(
+    paymentRequestHeaders: PaymentRequestHeaders,
+    paymentRequest: PaymentRequestDto,
+    paymentId: string,
+  ): Promise<string> {
+    const { senderEmail, companyName, logoUri } = defaultEmailConfig
+    const { receipientEmail, recipientName, paymentReason, paymentAmount, paymentCurrency, paymentExpiry } =
+      paymentRequest
+    const session = paymentRequestHeaders['reapit-session']
+    const clientCode = paymentRequestHeaders['reapit-customer']
+
+    const template = paymentRequestTemplate({
+      companyName,
+      logoUri,
+      paymentReason,
+      paymentCurrency: currencySymbolMapper(paymentCurrency),
+      url: `${process.env.PAYMENTS_APP_URI}/payments/${paymentId}?session=${session}&clientCode=${clientCode}`,
+      recipientName,
+      paymentExpiry: new Date(paymentExpiry).toDateString(),
+      paymentAmount: `${paymentAmount.toFixed(2)}`,
+    })
+    const title = `Payment Request from ${companyName}`
+
+    const mailId = await sendEmail(receipientEmail, title, template, senderEmail)
+
+    return mailId
```

Comment:
```suggestion
    await sendEmail(receipientEmail, title, template, senderEmail)
```

No need for declaration of unsed var, can return promise in async func